File: api/src/main/java/com/cinemo/api/domain/Ticket.java
package com.cinemo.api.domain;

import java.math.BigDecimal;
import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Ticket {
    private Long id;
    private Long bookingId; 
    private Long seatId;
    private String ticketCode;
    private BigDecimal price;
    private LocalDateTime issuedAt;
}

File: app/src/schemas/ticket.ts
import { z } from 'zod';

export const TicketResponseSchema = z.object({
  id: z.number(),
  bookingId: z.number(),
  seatId: z.number(),
  ticketCode: z.string(),
  price: z.number(),
  issuedAt: z.string(),
});

export const TicketListResponseSchema = z.object({
  bookingId: z.number(),
  totalTickets: z.number(),
  tickets: z.array(TicketResponseSchema),
});
